doku/static/src/store/types/actions.js

```diff
@@ -10,6 +10,7 @@ export const ADD_STYLESHEET_TO_TEMPLATE = 'addStylesheet';
 export const REMOVE_STYLESHEET_FROM_TEMPLATE = 'removeStylesheet';
 export const UPLOAD_STYLESHEET = 'uploadStylesheet';
 export const UPDATE_STYLESHEET = 'updateStylesheet';
+export const DELETE_STYLESHEET = 'deleteStylesheet';
```

Comment:
We use **remove** instead of **delete** in most places ;)

Reply:
Maybe we should overthink this as we are also *removing* stylesheets from templates without deleting them?